services/storage/src/simcore_service_storage/api/v0/openapi.yaml

```diff
@@ -208,6 +208,35 @@ paths:
                 $ref: '#/components/schemas/FileMetaDataArrayEnveloped'
         default:
           $ref: '#/components/responses/DefaultErrorResponse'
+  '/locations/{location_id}/files/{fileId}/s3/link':
```

Comment:
question: why not having a query parameter on ```/locations/{location_id}/files/{fileId}``` instead?
like:
```/locations/{location_id}/files/{fileId}?as_s3``` ?

Reply:
endpoint will be removed when integrating https://github.com/ITISFoundation/osparc-simcore/pull/3004